pkg/internal/wordpress/pod_template.go

```diff
@@ -339,7 +339,7 @@ func (wp *Wordpress) installWPContainer() []corev1.Container {
 	if len(wp.Spec.TLSSecretRef) > 0 {
 		scheme = "https"
 	}
-	url := fmt.Sprintf("%s://%s/", scheme, wp.Spec.Domains[0])
+	url := fmt.Sprintf("%s://%s/", scheme, wp.MainDomain())
```

Comment:
We can have a method called `wp.URL()`

Reply:
I don't think it's necessary.

Reply:
Just for the convenience and since we're re-using it a lot, I guess we can do it.

Reply:
Done